### PR TITLE
[BD-05] [TNL-7272] ORA v3 - Separate “Assessment Steps” and “Schedule” from Settings Tab

### PR DESCRIPTION
**Pull request by 0x29a**
# Description

This PR improves ORA Studio edit view:
1. Two new tabs added: `Schedule` and `Assessments Steps`.
2. Assessments settings moved to `Assessments Steps` tab from `Settings` tab.
3. All scheduling settings from `Settings` and `Assessments Steps` moved to `Schedule` tab.
4. Assessment settings now have new interface.

**JIRA tickets**:
- [TNL-7272](https://openedx.atlassian.net/browse/TNL-7272)
- [BLENDED-423](https://openedx.atlassian.net/browse/BLENDED-423)

**Discussions**: [old pull request](https://github.com/edx/edx-ora2/pull/1428).

**Dependencies**: None

**Screenshots**:

**Before:**

Screenshot_2020-07-08_18-38-06
Screenshot_2020-07-08_18-38-53

**After:**

Screenshot_2020-07-10_20-50-23
Screenshot_2020-07-10_20-50-41
Screenshot_2020-07-10_20-51-00
Screenshot_2020-07-10_20-51-21
Peek 2020-07-10 17-18

**Sandbox URL**: TBD - sandbox is being provisioned.

**Merge deadline**: None

**Testing instructions**:

1. Go to `$devstack_root/src`
2. Clone our ORA fork there (https://github.com/open-craft/edx-ora2)
3. Checkout to `0x29a/bb2502/split_settings_tab` branch.
4. Run `make install-local-ora`.
5. Add section, subsection and unit with ORA block to some course.
6. Open new unit and click `Edit`.

**Author notes and concerns**: None

**Reviewers**
- [ ] @giovannicimolin
- [ ] edX reviewer[s] TBD

### CODE REVIEW

**Comment by edx-webhook** [comment]
Thanks for the pull request, @0x29a! I've created [BLENDED-423](https://openedx.atlassian.net/browse/BLENDED-423) to keep track of it in Jira.

When this pull request is ready, tag your edX technical lead.

File: openassessment/xblock/studio_mixin.py [hunk 1]
```diff
@@ -159,7 +159,7 @@ def editor_context(self):
                 make_django_template_key(asmnt)
                 for asmnt in self.editor_assessments_order
             ],
-            'teams_feature_enabled': self.team_submissions_enabled,
+            'teams_feature_enabled': True,
```
Comment (giovannicimolin) [inline openassessment/xblock/studio_mixin.py:162]: @0x29a Is this a testing change? If so can you remove it from the final version?

Comment (0x29a) [reply openassessment/xblock/studio_mixin.py:162]: @giovannicimolin ah, yes, sure.

**Comment by 0x29a** [comment]
@giovannicimolin
>The staff and assessment deadlines in the schedules tab should appear between start and end dates, not below (if this isn't technically impossible or adds a lot of complexity).


I reordered deadline inputs:

Screenshot_2020-07-10_12-37-57

Is that what we want?

**Comment by 0x29a** [comment]
>The Specify start and due dates for the peer assessment step. label seems to be using a weird style, it is possible to improve it without modifying the code that much?

I've gotten style for it from `Rubric` tab:
Screenshot_2020-07-10_12-42-24

But I noticed that font for these labels on `Rubric` tab is slightly bigger than currently on `Schedule` tab. Do you mean this kind of improvement, or we should completely remake this label?

**Comment by 0x29a** [comment]
@giovannicimolin 
New markup of `Assessment Steps` tab:
Peek 2020-07-10 17-18

**Comment by marcotuts** [comment]
Hello! The proposed UI for the Assessment Steps looks great, thanks for the helpful visual / recording. 

As for the schedule tab is that something we can adjust slightly? I had a few questions for this tab: 


1.) Can we add headings between each pair of Start Date + Due Date fields to make clearer which schedule settings correspond to which assessment step? 
2.) Can it only show schedule / date settings for enabled steps? 
3.) Can the order of the paired Start Date + Due Date for each step echo the configuration order for that problem or is it currently not responding to the current (or mid-edit) state of the assessment steps and their order. This I could imagine being complex but I'm not sure how possible it is to have the schedule be in order of step progression from the assessment steps area.


These questions are ordered in terms of importance, of 3 is a lot of work to even partially achieve we can cut for now (or put into a future ticket regardless of its size!) Ideally 1 + 2 we would be able to address, but 1 is the more critical of the two. 

@0x29a  thoughts on this? Thanks!

**Comment by 0x29a** [comment]
>1.) Can we add headings between each pair of Start Date + Due Date fields to make clearer which schedule settings correspond to which assessment step?

That's reasonable. I'll add headings and update screenshots in the PR description.

>2.) Can it only show schedule / date settings for enabled steps?

Ah, yes, there are no screenshots showing it, but actually that's how it works now.

> 3.) Can the order of the paired Start Date + Due Date for each step echo the configuration order for that problem or is it currently not responding to the current (or mid-edit) state of the assessment steps and their order. This I could imagine being complex but I'm not sure how possible it is to have the schedule be in order of step progression from the assessment steps area.

I think this shouldn't be super hard to implement, but probably it's better to do in future ticket, since diff for this pull requests is already huge. I'll coordinate with @giovannicimolin about the best way to proceed.

Thanks for checking this out, @marcotuts!

**Comment by 0x29a** [comment]
I'm experiencing some troubles with sandbox creation, but this is anyway ready for edX code review.

**Comment by 0x29a** [comment]
Thanks for review, @davestgermain.

I rebased branch, bumped version to `2.9.0` and squashed commits.

**Pull request merged**